.strigo/config.yml

```diff
@@ -1,45 +1,88 @@
-      # `file` is relative to .strigo directory
+  - lab_resources:
+      - display_name: "AWS EC2 Instance"
+        server_image_id: "ami-0abcdef1234567890"
+        platform_type: "lab"
+        image_platform: "linux"
+        cloud_provider: "aws"
+        view_interface: "Desktop"
+        instance_type: "t2.micro"
+      - display_name: "Web Resource"
+        web_resource_url: "https://example.com"
+        platform_type: "web"
+    exercises:
       - file: "../hello-world/hello.md"
         title: "Hello!"
       - file: "../hello-world/world.md"
         title: "World!"
 
-## Class exercises - configuration for multiple classes
+## Class lab resources - configuration for multiple classes
 # ---
 # classes:
-#   # `class_id` may be omitted if there is only one class
-#   # in this configuration file
 #   - class_id: "<your_class_id>"
+#     lab_resources:
+#       - display_name: "AWS EC2 Instance"
+#         server_image_id: "ami-0abcdef1234567890"
+#         platform_type: "lab"
+#         image_platform: "linux"
+#         cloud_provider: "aws"
+#         view_interface: "Desktop"
 #     exercises:
 #       - file: "../hello-world/hello.md"
 #         title: "Hello!"
 #   - class_id: "<your_class_id>"
+#     lab_resources:
+#       - display_name: "Web Resource"
+#         web_resource_url: "https://example.com"
+#         platform_type: "web"
 #     exercises:
 #       - file: "../hello-world/world.md"
 #         title: "World!"
 
-## Class exercises - configuration for different supported syntaxes
+## Class lab resources - configuration for different resource types
 # ---
 # classes:
-#   - exercises:
-#       - file: "../markup-syntaxes/markdown.md"
-#         # `syntax` is optional
-#         # can be "md" | "md-extended" | "rst"
-#         # uses file extension as fallback
-#         syntax: "md"
-#         title: "Markdown!"
-#       - file: "../markup-syntaxes/markdown-extended.md"
-#         syntax: "md-extended"
-#         title: "Extended markdown"
-#       - file: "../markup-syntaxes/restructured.rst"
-#         syntax: "rst"
-#         title: "Restructed"
-#       - file: "../relative-images/md-with-relative-imgs.md"
-#         title: "Relative images"
+#   - lab_resources:
+#       - display_name: "AWS EC2 Instance"
+#         server_image_id: "ami-0abcdef1234567890"
+#         platform_type: "lab"
+#         image_platform: "linux"
+#         cloud_provider: "aws"
+#         view_interface: "Desktop"
+#       - display_name: "Azure VM"
+#         azure_vm_definition:
+#           location: "eastus"
+#           image_publisher: "Canonical"
+#           image_offer: "UbuntuServer"
+#           image_sku: "18.04-LTS"
+#           image_version: "latest"
+#           custom_username: "azureuser"
+#         platform_type: "lab"
+#         image_platform: "linux"
+#         cloud_provider: "azure"
+#         view_interface: "Terminal"
+#       - display_name: "Licensed AWS Product"
+#         aws_licensed_product:
```

Comment:
the aws_licensed_product structure is for internal use.
the users shouldn't really provide the product id and name.
instead- let's just have a boolean field for indicating the usage of the MS Office licenses.
something like `include_ms_office_licence` / `is_ms_office_licensed`